src/components/pages/LandingPage.vue

```diff
@@ -29,8 +46,10 @@ export default defineComponent({
       <SignSection />
     </TheHeader>
     <WelcomeSection />
-    <MoviesSummary />
-    <ScreeningsFilters />
+    <TheContainer>
```

Comment:
What is doing `TheContainer` here? Im just curious 🙈

Reply:
It simply keeps padding of the content, where needed 😄

Reply:
If it is only for wrapping purposes i think div is just enough. Using another component suggest that it does something more 👌 Have you figured out why there are changes from another branch 🙈

Reply:
I have reviewed the whole file on remote and TheContainer is where it should be, sorry for the confusion. Actually it was Szymon to suggest us this kind of component and I liked the idea and applied it.